modules/tutorials/pages/java-tutorial/second-couchbase-application.adoc

```diff
@@ -0,0 +1,124 @@
+:page-pagination:
+
+= Your second Couchbase application: populating the course details
+
+[abstract]
+Your first application created a single student record for the student collection. In this part, you're going to populate the course collection.
+
+
+== Populating the course details collection
+
+You can use the same technique to build a store for the courses. Here's a quick reminder of the course document structure:
+
+[tabs]
+====
+art history::
++
+--
+[source, json]
+----
+include::partial$art-history-course.json[]
+----
+--
+
+graphic design::
++
+--
+[source, json]
+----
+include::partial$graphic-design-course.json[]
+----
+--
+
+fine art::
++
+--
+[source, json]
+----
+include::partial$fine-art-course.json[]
+----
+--
+====
+
+[NOTE]
+====
+The documents include an additional `type` field. Do you remember why we include a `type` for all our documents?
+
+.Answer
+[%collapsible]
+=====
+Documents of different types can reside in the same bucket/collection, so it's useful to include a field which can be used to filter documents by type.
```

Comment:
This jarred a little here - we're explicitly posting these documents to the `course-record-collection` collection, so we don't really need a type?

Do we need an alternative example to contrast better with scope/collection usage?

Reply:
Yup, I did think about this, but I think it's more important that folk use the 'type' field to shield themselves from problems later on. I'm the same way with primary fields: you should always have a primary field that isn't related to the data so you can do stuff later on, like change account numbers.